src/components/ChatEntry.js

```diff
@@ -1,22 +1,46 @@
-import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const onLikeButtonClick = () => {
+    const updateChatEntry = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+    };
+    props.onUpdateLiked(updateChatEntry);
+  };
+
+  const heart = props.liked ? '❤️' : '🤍';
+  const alignment = props.sender === 'Vladimir'? 'chat-entry local' : 'chat-entry remote';
+  const bubbleAlign = props.sender === 'Vladimir' ? 'entry-bubble local' : 'entry-bubble remote';
```

Comment:
A few things to consider:

- How could we dynamically get the names of the chat participants? What if our chat data were to change and the messages were between Mario and Luigi (or -literally any other name lol) ?
- What if the chat were just starting and there weren't any messages yet?